src/common/url/parse-url.js

```diff
@@ -25,13 +25,14 @@ export function parseUrl (url) {
 
   if (isBrowserScope) {
     // Use an anchor dom element to resolve the url natively.
-    urlEl = document.createElement('a');
-    urlEl.href = url;
-  } else {
+    urlEl = document.createElement('a')
+    urlEl.href = url
+  }
+  else {
     try {
-      urlEl = new URL(url, location.href);
-    } catch {
-      return ret; // IE doesn't support URL(), so for IE non-web env (e.g. workers), this func will break and return {}
+      urlEl = new URL(url, location.href)
+    } catch (err) {
+      return ret
```

Comment:
Why remove the comment? We may not support IE, but it doesn't hurt knowing customers still use it.

Reply:
https://issues.newrelic.com/browse/NEWRELIC-6479